tests/behaviour/concept/type/attributetype/attribute_type_steps.py

```diff
@@ -0,0 +1,148 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+from behave import *
+from hamcrest import *
+
+from grakn.concept.type.value_type import ValueType
+from tests.behaviour.config.parameters import parse_value_type, parse_list
+from tests.behaviour.context import Context
+
+
+@step("put attribute type: {type_label}, with value type: {value_type}")
+def step_impl(context: Context, type_label: str, value_type: str):
+    context.tx().concepts().put_attribute_type(type_label, parse_value_type(value_type))
+
+
+@step("attribute({type_label}) get value type: {value_type}")
+def step_impl(context: Context, type_label: str, value_type: str):
+    assert_that(context.tx().concepts().get_attribute_type(type_label).get_value_type(), is_(parse_value_type(value_type)))
+
+
+@step("attribute({type_label}) get supertype value type: {value_type}")
+def step_impl(context: Context, type_label: str, value_type: str):
+    supertype = context.tx().concepts().get_attribute_type(type_label).as_remote(context.tx()).get_supertype()
+    assert_that(supertype.get_value_type(), is_(parse_value_type(value_type)))
+
+
+def attribute_type_as_value_type(context: Context, type_label: str, value_type: ValueType):
+    attribute_type = context.tx().concepts().get_attribute_type(type_label)
+    if value_type is ValueType.OBJECT:
```

Comment:
`OBJECT` value type is unfamiliar to me - what is it?

Reply:
`OBJECT` is a value type that root `attribute` can have. When retrieving root `attribute` with value type `OBJECT` and calling `getSubtypes`, attribute types of all value types are retrieved.